components/style/properties/data.py

```diff
@@ -63,12 +72,20 @@ def maybe_cast(self, type_str):
         return "as " + type_str if self.needs_cast() else ""
 
 
+def arg_to_bool(arg):
+    if isinstance(arg, bool):
+        return arg
+    else:
+        assert arg == "True" or arg == "False"
```

Comment:
nit: No need for the `else` branch, you can just dedent this block. Also, `assert arg in ["True", "False"]`?